compiler_opt/tools/extract_ir.py

```diff
@@ -59,11 +60,15 @@
     'Include only those modules with a command line matching this regexp. '
     'Setting it to None for not filtering. Note that the regexp is applied '
     'independently for each separate command line option. For example, ^-Oz$ '
-    'will match Oz - built binaries.')
-flags.DEFINE_bool(
-    'thinlto_build', False, 'Set if the build was ThinLTO, to '
-    'ensure index files are also copied. The build is assumed to have had'
-    '-mllvm -lto-embed-bitcode=post-merge-pre-opt passed to clang.')
+    'will match Oz - built binaries. Does not work with thinlto_build=lld.')
+flags.DEFINE_enum(
```

Comment:
I think it'd be better to put `distributed` and `local`? Technically both are done in-process. Also I believe she's OOO

Reply:
wait actually nothing's stopping you from running ThinLTO with `clang` on a local machine either, so it might be best to just leave it as clang vs lld?

Reply:
it's about the way thinlto is designed to be used: either within lld, or the distributed build way. You're right that one could technically just use 1 clang invocation and pass a bunch of .cpp files and also do thinlto, but that's not the point - that clang (acting as a driver) will still spawn lld and enact the `local` scenario. (if I understood your comment correctly)

`distributed` and `local` sgtm!

Reply:
oh I meant doing `lld -thinlto-index-only` and then `clang -fthinlto-index=x.thinlto.bc -x -ir a.bc` on a local machine rather than sending out the files to remote machines to run. I agree it's not the intended way to use ThinLTO though so we should be fine with distributed/local